docs/programmatic.rst

```diff
@@ -93,3 +93,17 @@ The returned slice is an Astropy :class:`~astropy.io.fits.PrimaryHDU` instance,
 which you can write to disk using::
 
     >>> slice1.writeto('my_slice.fits')  # doctest: +SKIP
+
+
+Retrieving the Pixel Coordinates
+--------------------------------
+
+With a path in hand, you can recover the pixel coordinates used in the
+extraction::
+
+    >>> from spectral_cube import SpectralCube
+    >>> cube = SpectralCube.read('my_cube.fits')  # doctest: +SKIP
```

Comment:
Is there a hosted spectral cube file that we can use in general so the example are tested?

Reply:
Maybe we should use the L1448 cube from the example?

Reply:
👍

Reply:
done